NonWeb.md

```diff
@@ -20,8 +21,19 @@ both browsers and node.js. This situation is expected to be first encountered
 with POSIX features such as file I/O. In that respect, WebAssembly would err
 towards standardizing existing practice through libraries, and let developers
 choose which libraries to use.
+Similarly, the WebGL and SDL interface (
+[exposed by emscripten](http://kripken.github.io/emscripten-site/docs/porting/multimedia_and_graphics/OpenGL-support.html)
+in C/C++) could be made into a shared spec with non-browser mobile/desktop
+environments, allowing for cross platform browser & non-browser games and other
+things.
 
 The WebAssembly spec will not try to define any large portable libc-like
 library. However, certain features that are core to WebAssembly semantics that
 are found in native libc *would* be part of the core WebAssembly spec as either
 primitive opcodes or a special builtin module (e.g., `sbrk`, `mmap`).
+
+In general, by keeping the non-browser path such that it doesn't require too
+much "browser emulation" (JS/DOM dependencies), WebAssembly could be used as
```

Comment:
I read "JS" to imply ECMA, which doesn't have any special tie to the browser for cases like node.js. The "browser emulation" would be more, for example, what's defined by WhatWG. May be insignificant at this level of detail but wanted to bring it up.

May possibly want to separate how JS impacts wasm (e.g. need for GC) and how the browser APIs do the same (e.g. WebWorkers, event loop, I/O).